aicspylibczi/CziFile.py

```diff
@@ -262,21 +262,22 @@ def read_image(self, m_index: int = -1, **kwargs):
 
         Parameters
         ----------
-        m_index
+        m_index : int (optional)
```

Comment:
Could someone potentially pass both `m_index` and `M`? Would there be conflicts?

Reply:
It _may_ be worth it to simply expand all these from kwargs to full parameters. Your choice though.